java/src/main/native/src/ScalarJni.cpp

```diff
@@ -15,9 +15,11 @@
  */
 
 #include <cudf/binaryop.hpp>
+#include <cudf/fixed_point/fixed_point.hpp>
 #include <cudf/scalar/scalar_factories.hpp>
 
 #include "cudf_jni_apis.hpp"
+#include "dtype_utils.hpp"
```

Comment:
Please note that `dtype_utils.hpp` is not independently usable in its current form. You can verify this by making it the first `#include` in `ScalarJni.hpp`. It works now because it is included after the other `cudf` headers.
Please `#include <cudf/types.hpp>` and the headers for any other symbols you need in `dtype_utils.hpp`.

Reply:
Thanks for incorporating this suggestion.